sheriff.go

```diff
@@ -60,8 +60,6 @@ func Marshal(options *Options, data interface{}) (interface{}, error) {
 		options.nestedGroupsMap = make(map[string][]string)
 	}
 
-	checkGroups := len(options.Groups) > 0
```

Comment:
Another minor change-- if you don't specify any groups, we'll filter out anything that has groups specified.